### ISSUE TITLE
Error: Expecting terminator

### ISSUE THREAD

**Issue by wnsnty** [seishun/node-steam-resources#4]
````
Error: Expecting terminator

    at expect (\node_modules\steam-resources\steam_language_parser\parser\token_analyzer.js:188:11)

    at parseInnerScope (\node_modules\steam-resources\steam_language_parser\parser\token_analyzer.js:161:7)

    at Object.exports.analyze (\node_modules\steam-resources\steam_language_parser\parser\token_analyzer.js:103:15)

    at Object.exports.analyze (\node_modules\steam-resources\steam_language_parser\parser\token_analyzer.js:36:33)

    at Object.exports.analyze (\node_modules\steam-resources\steam_language_parser\parser\token_analyzer.js:36:33)

    at Object.<anonymous> (\node_modules\steam-resources\steam_language_parser\index.js:12:27)

    at Module._compile (module.js:635:30)

    at Object.Module._extensions..js (module.js:646:10)

    at Module.load (module.js:554:32)

    at tryModuleLoad (module.js:497:12)
````

This occurs even when 

`var Steam = require('steam-resources');`

is the only thing in the file.

**Comment by matheuschagas** [comment]
any news?

**Comment by essramos** [comment]
I am having the same issue.

**Comment by walamana** [comment]
Any updates yet?

**Comment by seishun** [comment]
Fixed in 893606cba07885f4b4ca39817ad1ba92d5483c3e.
